### PR TITLE
CustomerIdentity: Create RemoveCustomer Req/Res Mappings

### PR DESCRIPTION
**Pull request by billpull**

### CODE REVIEW

File: src/GIDX.SDK/ICustomerIdentityClient.cs [hunk 1]
```diff
@@ -51,5 +51,12 @@ public interface ICustomerIdentityClient : IClient
         /// <param name="request"></param>
         /// <returns></returns>
         LocationResponse Location(LocationRequest request);
+
+        /// <summary>
+        /// Make a request to our RemoveCustomer endpoint.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        RemoveCustomerResponse Location(RemoveCustomerRequest request);
```
Comment (nicklmiller) [inline src/GIDX.SDK/ICustomerIdentityClient.cs:60]: Method name should be RemoveCustomer, not Location.

File: src/GIDX.SDK/Models/CustomerIdentity/RemoveCustomerRequest.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GIDX.SDK.Models.CustomerIdentity
+{
+    public class RemoveCustomer : RequestBase
```
Comment (nicklmiller) [inline src/GIDX.SDK/Models/CustomerIdentity/RemoveCustomerRequest.cs:9]: Class name should be RemoveCustomerRequest, not RemoveCustomer.

**Comment by nicklmiller** [comment]
Hey Bill, I really appreciate you adding a pull request for this. I'm going to be upgrading our SDK to .NET Core and adding our new Direct Cashier. I'll make sure to get RemoveCustomer added too.

This pull request doesn't build for me, however. The ICustomerIdentityClient interface method is incorrect and the RemoveCustomerRequest class is not named properly.

**Comment by billpull** [comment]
@nicklmiller thanks for the update. In a related question is there a CustomerIdentity API method for updating a CustomerProfile with more data? Based on the docs you can only call Register again.

**Comment by nicklmiller** [comment]
No, there is not an API for that. You can call CustomerRegistration again, but that would do a full verification again as well.

**Pull request closed without merging**